descope/flask/__init__.py

```diff
@@ -245,9 +245,52 @@ def decorated(*args, **kwargs):
     return decorator
 
 
-def descope_verify_code_by_whatsapp(descope_client: DescopeClient):
+def descope_verify_code_by_phone_voice_call(descope_client: DescopeClient):
```

Comment:
Did you add an example that initiates the phone call ?

Reply:
These are examples funcs, so added also for voice, and fixed names.